environment/peercontainer.go

```diff
@@ -169,6 +169,13 @@ func (e *PeerContainer) Destroy() error {
 
 // Download downloads the path to outside the environment
 func (e *PeerContainer) Download(path string) (string, error) {
+	fileInfo, err := e.Stat(path)
+	if err != nil {
+		return path, fmt.Errorf("failed to stat the given path : %s. Error: %v", path, err)
+	}
+	if !fileInfo.IsDir() {
+		return path, fmt.Errorf("download only supports directory paths. The path provided is %s. Error: %v", path, err)
```

Comment:
Is there a reason to support only directories? Or is our current implementation support only directories?

Reply:
@ashokponkumar From what I understand, I think it is for directories as of now. If you see the containerized transformer code, we actually create a tmp folder to upload the input json.

Having the upload and download path to include files would be a good flexibility to have.